.github/actions/setup-binary-builds/action.yml

```diff
@@ -127,15 +127,21 @@ runs:
           CONDA_ENV="${RUNNER_TEMP}/conda_environment_${GITHUB_RUN_ID}"
           export CONDA_EXTRA_PARAM=""
 
-          if [[ ( "${PYTHON_VERSION:-}" == *t || "${PYTHON_VERSION:-}" == 3.14* ) && "$(uname)" == Darwin ]]; then
-            # downgrade conda version for python 3.13t/3.14t install.
-            # TODO: remove this once python 3.13t/3.14t is fully supported on conda
+          if [[ ( "${PYTHON_VERSION:-}" == *t || "${PYTHON_VERSION:-}" == 3.14* ) ]]; then
+
+            # downgrade conda version for python 3.13t install.
+            # TODO: remove this once python 3.13t is fully supported on conda
```

Comment:
The previous comment stated that it was needed for both 3.13t and 3.14t but the new comment seems to drop the 3.14t version should it still include 3.14t in the comment considering the if statement seems to pick it up?

Reply:
Fixed the comment